bin/plot_krun_results

```diff
@@ -766,16 +782,7 @@ def get_data_dictionaries(json_files, benchmarks=[], wallclock_only=False,
                         data_dictionary['aperf_counts'][key][machine] = data['aperf_counts'][key]
```

Comment:
Just above here, line 759 is the switch `if benchmarks == []:` and that's the if/else I suggest we remove.